### PR TITLE
add convenient assertSoftly for arbitrary commands  

### PR DESCRIPTION
**Pull request by qoomon**
#### Check List:
* Unit tests : YES
* Javadoc with a code example (API only) : YES 

simplified approach of PR #1460 (Add method to use SoftAssertions with any custom Assertions)

### CODE REVIEW

**Comment by joel-costigliola** [comment]
Thanks, we will look at it after releasing 3.13.0.

**Comment by qoomon** [comment]
Nice, basically it provides the same functionality as `org.junit.jupiter.api.Assertions.assertAll(...)` from jUnit5

**Comment by qoomon** [comment]
Maybe it make sense to add the functionality to pass multiple conditions(varargs) to all `satisfy` methods as well. Implementation is the done by `assertAll(conditions)`

File: src/test/java/org/assertj/core/api/BDDSoftAssertionsTest.java [hunk 1]
```diff
@@ -873,6 +873,17 @@ public void should_assert_using_assertSoftly() {
        .hasMessageContaining("blue");
   }
 
+  @Test
+  public void should_assert_using_assertSoftlyExecutables() {
```
Comment (joel-costigliola) [inline src/test/java/org/assertj/core/api/BDDSoftAssertionsTest.java]: is there a difference with the test you have added in `SoftAssertionsTest` ? it there is not remove this one in favor of the other.

**Comment by qoomon** [comment]
Sorry for the long delay

**Pull request closed without merging**